docs/.vuepress/config.js

```diff
@@ -167,6 +167,10 @@ module.exports = {
         app: ['./docs/.vuepress/public-path.js', config.entry.app[0]]
       }
     }
+    config.module.rules.push({
+      test: /\.js$/,
+      loader: require.resolve('@open-wc/webpack-import-meta-loader')
```

Comment:
Webpack does not yet support [`import.meta`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta) but the drand client is using this feature to know where to download the wasm from. Using `import.meta` means that when you use the library in the browser with esmodules like this...:

```html
<script type="module">
import Client, { HTTP } from 'https://cdn.jsdelivr.net/npm/@alanshaw/drand-client@0.1.0-pre.3/drand.js'
// ...
</script>
```

The WASM will be downloaded from `https://cdn.jsdelivr.net/npm/@alanshaw/drand-client@0.1.0-pre.3/wasm/drand.wasm` and not the domain you're hosting the site on.

Similarly, in Node.js `import.meta` allows us to read files relative to the file that is being executed (`__dirname` is not available in esmodules).

Since we have to work with vuepress on the drand website we have to fix [the issue with webpack](https://github.com/webpack/webpack/issues/6719).